e2e/tests/functional/plugins/imagery/exampleImagery.e2e.spec.js

```diff
@@ -746,6 +747,28 @@ async function performImageryViewOperationsAndAssert(page, layoutObject) {
   await page.clock.resume();
   await expect(page.getByLabel(lastImageTimestamp)).toBeHidden();
 
+  // go way forward in time to ensure multiple images are discarded
+  const IMAGES_TO_DISCARD_COUNT = 5;
```

Comment:
this is fantastic

Reply:
(this is supposed to be for the whole test part)

Reply:
its okay. I know my `const`s are internationally famous

Reply:
Dave "Some of the Best Consts" Tsay